docs/geedocs/docsrc/answer/getStarted/download.rst

```diff
@@ -8,9 +8,11 @@ Download documentation for GEE 5.1.x
 
    .. container:: content
 
+      .. index:: Download Documentation
+
       The majority of Google Earth Enterprise help content is now
       maintained in our `online
-      documentation <http://www.opengee.org/geedocs/5.2.1/>`__, which
+      documentation <http://www.opengee.org/geedocs/5.2.1/>`_, which
```

Comment:
This is pointing specifically to the 5.2.1 docs. Maybe update this to the page that allows you to select the version.